ros/src/twist_controller/yaw_controller.py

```diff
@@ -10,16 +11,21 @@ def __init__(self, wheel_base, steer_ratio, min_speed, max_lat_accel, max_steer_
         self.min_angle = -max_steer_angle
         self.max_angle = max_steer_angle
 
-
     def get_angle(self, radius):
         angle = atan(self.wheel_base / radius) * self.steer_ratio
         return max(self.min_angle, min(self.max_angle, angle))
 
     def get_steering(self, linear_velocity, angular_velocity, current_velocity):
-        angular_velocity = current_velocity * angular_velocity / linear_velocity if abs(linear_velocity) > 0. else 0.
```

Comment:
Is it really needed to go away from default implementation, did you find some improvements after this?

Reply:
It was done to make the code more comprehensible. As a general rule, you should avoid expressions lengthily in the code, especially if their length exceeds the conventional line length limits (80 characters per line for 1990th-2000th and now it is about 120 characters per line).
![screenshot from 2019-01-14 00-56-22](https://user-images.githubusercontent.com/2496083/51091138-4d8b8300-1797-11e9-95c7-552a9cbb5910.png)
).

The logic was not changed at all. 

The same applies to the code a few lines below. I was just following IDE's suggestions on code style improvements.